paas_app_charmer/app.py

```diff
@@ -259,6 +259,17 @@ def map_integrations_to_env(integrations: IntegrationsState, prefix: str = "") -
             if v is not None
         )
 
+    if integrations.rabbitmq_parameters:
+        rabbitmq = integrations.rabbitmq_parameters
+        env.update(
+            (
+                ("RABBITMQ_HOSTNAME", rabbitmq.hostname),
+                ("RABBITMQ_USERNAME", rabbitmq.username),
+                ("RABBITMQ_PASSWORD", rabbitmq.password),
+                ("RABBITMQ_VHOST", rabbitmq.vhost),
+            )
```

Comment:
Shouldn't we also provide the settings in URI format ? "amqp://...."

Reply:
That is a very insightful! I am going to replace our own structure with a URI/URL. Thanks!

Reply:
I have changed some of the code to use a amqp uri instead of the parameter. https://www.rabbitmq.com/docs/uri-spec